client/.dockerignore

```diff
@@ -0,0 +1,8 @@
+build
+node_modules
+.dockerignore
+.eslintignore
+.eslintrc.json
+.gitignore
+.nvmrc
+README.md
```

Comment:
Instead of blacklisting you could use whitelisting:
```
*
!config
!public
!src
!package.json
!tsconfig.json
!yarn.lock
```
But that's just a matter of preference.